repotracker/repotracker.go

```diff
@@ -589,3 +595,70 @@ func createVersionItems(v *version.Version, ref *model.ProjectRef, project *mode
 	}
 	return nil
 }
+
+func addBuildBreakSubscriptions(v *version.Version, projectRef *model.ProjectRef) error {
+	if !projectRef.NotifyOnBuildFailure {
+		return nil
+	}
+	subscriptionBase := event.Subscription{
+		Type:      event.ResourceTypeVersion,
+		Trigger:   "regression",
+		Selectors: trigger.MakeVersionSelectors(*v),
+	}
+	subscribers := []event.Subscriber{}
+
+	// if the commit author wants build break notifications, don't send to admins
+	if v.AuthorID != "" {
+		subscriber, _ := makeBuildBreakSubscriber(v.AuthorID)
+		if subscriber != nil {
+			return nil
+		}
+	}
+	// if the project has build break notifications, subscribe admins if no one subscribed
+	catcher := grip.NewSimpleCatcher()
+	if projectRef.NotifyOnBuildFailure && len(subscribers) == 0 {
```

Comment:
len(subscribers) will always be 0 as written.

Reply:
fixed